modules/arm_plugin/armplg_build.sh

```diff
@@ -33,23 +33,23 @@ checkSrcTree()
 {
     [ $# -lt 2 ] && fail
 
-    TEST_DIR_NAME=directory
+    BRANCH_NAME=master
     if [ $# -gt 2 ]; then
-        TEST_DIR_NAME="$3 directory"
```

Comment:
What do you think about making third parameter mandatory?

Reply:
Agree, fixed